### ISSUE TITLE
Recurrent reservations won't work

### ISSUE THREAD

**Issue by Loiseau2nuit** [glpi-project/glpi#15182]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.7

### Bug description

Hello there,

No luck in the french support section with this, so i'm hoping someone will be able to help here :

I'm desperately trying to deal with equipment recurrent reservations, for our colleagues working remotely at least one day per week.

New test made today with no succes :

Reserve an item
- item "computer - XXXXX"
- by : John Doe
- start date : 05/24/2023 (wednesday)
- Duration :  1 day
- Repetition : weekly
- end date : 07/15/2023 (no remote work during summer vacations)
- Per day : wednesday

I click 'Add', the first reservation will be created accordingly, but the following occurences won't, and for each, the same error message displays : "Error while selecting dates. Start date is more recent than end date" which, at this point doesn't make any sense, considering every dates I entered are actually in the FUTURE ...

### Relevant log output

```shell
Some console output while trying this feature:
----------------------------------------------

JQMIGRATE: Migrate is installed with logging active, version 3.4.0 base.js:103:17


downloadable font: no supported format found (font-family: "tabler-icons" style:normal weight:400 stretch:100 src index:1) source: (end of source list)


jQuery.Deferred exception: templateResult is not defined @https://glpi.************.**/front/reservation.form.php:362:13
mightThrow@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3790:29
__webpack_modules__</</Deferred/then/resolve/</process<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3858:12
setTimeout handler*__webpack_modules__</</Deferred/then/resolve/<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3896:16
fire@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3524:31
fireWith@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3654:7
fire@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3662:10
fire@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3524:31
fireWith@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3654:7
ready@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:4134:13
completed@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:4144:9
EventListener.handleEvent*__webpack_modules__</<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:4160:11
__webpack_modules__</<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:52:4
__webpack_modules__<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:64:4
__webpack_require__@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62677:42
@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62800:28
@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62889:3
@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62891:12
	undefined jquery.js:4050


jQuery.Deferred exception: templateResult is not defined @https://glpi.************.**/front/reservation.form.php:1532:13
mightThrow@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3790:29
__webpack_modules__</</Deferred/then/resolve/</process<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3858:12
setTimeout handler*__webpack_modules__</</Deferred/then/resolve/<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3896:16
fire@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3524:31
fireWith@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3654:7
fire@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3662:10
fire@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3524:31
fireWith@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:3654:7
ready@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:4134:13
completed@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:4144:9
EventListener.handleEvent*__webpack_modules__</<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:4160:11
__webpack_modules__</<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:52:4
__webpack_modules__<@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:64:4
__webpack_require__@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62677:42
@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62800:28
@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62889:3
@https://glpi.************.**/public/lib/base.js?v=5eaa44ad76ced0de9f3b57cdb0268aa4a8b52676:62891:12
	undefined jquery.js:4050


Uncaught ReferenceError: templateResult is not defined
	 	 <anonymous> reservation.form.php:362
	 	 jQuery 13
	 	 Webpack 4
reservation.form.php:362:13


Uncaught ReferenceError: templateResult is not defined
	 	 <anonymous> reservation.form.php:1532
	 	 jQuery 13
	 	 Webpack 4
reservation.form.php:1532:13


JQMIGRATE: Migrate is installed with logging active, version 3.4.0 base.js:103:17
Uncaught TypeError: $(...).sortable is not a function
	 	 <anonymous> https://glpi.************.**/plugins/reservation/js/tools.js?v=310c5750b1c1b221655034bc868c9a46ef2b4cbe:141
	 	 <anonymous> https://glpi.************.**/plugins/reservation/js/tools.js?v=310c5750b1c1b221655034bc868c9a46ef2b4cbe:150
tools.js:141:21
	 	 <anonyme> https://glpi.************.**/plugins/reservation/js/tools.js?v=310c5750b1c1b221655034bc868c9a46ef2b4cbe:141
	 	 <anonyme> https://glpi.************.**/plugins/reservation/js/tools.js?v=310c5750b1c1b221655034bc868c9a46ef2b4cbe:150


L’utilisation du capteur d’orientation n’est plus permise. libs.js:13:2141


downloadable font: no supported format found (font-family: "tabler-icons" style:normal weight:400 stretch:100 src index:1) source: (end of source list)


Plugin file not loaded for resourceTimeline fullcalendar.js:4566:29
```

### Steps To reproduce

Reservation > Reserve an item
- item "computer - XXXXX"
- by : John Doe
- start date : 05/24/2023 (wednesday)
- Duration :  1 day
- Repetition : weekly
- end date : 07/15/2023 (no remote work during summer vacations)
- Per day : wednesday

### Your GLPI setup information

<details><summary>Informations sur le système, l'installation et la configuration</summary><pre>GLPI 10.0.7-git-bce21331d5-bce21331d5 ( => /var/www/glpi)
Installation mode: GIT
Current language:fr_FR

</pre></details><details><summary>Server</summary><pre>
Operating system: Linux uvappp05 5.10.0-23-amd64 #1 SMP Debian 5.10.179-1 (2023-05-12) x86_64
PHP 8.2.6 fpm-fcgi (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apcu, bz2, calendar, cgi-fcgi, ctype, curl,
	date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imagick, imap, intl, json, ldap, libxml, mbstring, mysqli,
	mysqlnd, openssl, pcre, pdo_mysql, posix, random, readline, session, shmop, soap, sockets, sodium, standard, sysvmsg, sysvsem,
	sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: nginx/1.18.0
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36
Server Software: Debian 11
	Server Version: 10.5.19-MariaDB-0+deb11u2
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: GLPIdbAdmin@localhost/glpi
	Host info: Localhost via UNIX socket
	
PHP version (8.2.6) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.5.19) is supported.
No files from previous GLPI version detected.
The log file has been created successfully.
Write access to /var/lib/glpi/_cache has been validated.
The directory could not be created in /etc/glpi/.
Write access to /var/lib/glpi/_cron has been validated.
Write access to /var/lib/glpi has been validated.
Write access to /var/lib/glpi/_dumps has been validated.
Write access to /var/lib/glpi/_graphs has been validated.
Write access to /var/lib/glpi/_lock has been validated.
Write access to /var/lib/glpi/_pictures has been validated.
Write access to /var/lib/glpi/_plugins has been validated.
Write access to /var/lib/glpi/_rss has been validated.
Write access to /var/lib/glpi/_sessions has been validated.
Write access to /var/lib/glpi/_tmp has been validated.
Write access to /var/lib/glpi/_uploads has been validated.

Web server root directory configuration seems safe.
Sessions configuration is secured.
OS and PHP are relying on 64 bits integers.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/glpi/marketplace has been validated.
Timezones seems loaded in database.

</pre></details><details><summary>GLPI constants</summary><pre>
GLPI_ROOT: "/var/www/glpi"
GLPI_CONFIG_DIR: "/etc/glpi/"
GLPI_VAR_DIR: "/var/lib/glpi"
GLPI_LOG_DIR: "/var/log/glpi"
GLPI_MARKETPLACE_DIR: "/var/www/glpi/marketplace"
GLPI_USE_CSRF_CHECK: "1"
GLPI_CSRF_EXPIRES: "7200"
GLPI_CSRF_MAX_TOKENS: "100"
GLPI_USE_IDOR_CHECK: "1"
GLPI_IDOR_EXPIRES: "7200"
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: false
GLPI_SERVERSIDE_URL_ALLOWLIST: ["/^(https?|feed):\\/\\/[^@:]+(\\/.*)?$/"]
GLPI_TELEMETRY_URI: "https://telemetry.glpi-project.org"
GLPI_INSTALL_MODE: "GIT"
GLPI_NETWORK_MAIL: "glpi@teclib.com"
GLPI_NETWORK_SERVICES: "https://services.glpi-network.com"
GLPI_MARKETPLACE_ALLOW_OVERRIDE: true
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: true
GLPI_USER_AGENT_EXTRA_COMMENTS: ""
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: "1"
GLPI_AJAX_DASHBOARD: "1"
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: "0"
GLPI_CENTRAL_WARNINGS: "1"
GLPI_DOC_DIR: "/var/lib/glpi"
GLPI_CACHE_DIR: "/var/lib/glpi/_cache"
GLPI_CRON_DIR: "/var/lib/glpi/_cron"
GLPI_DUMP_DIR: "/var/lib/glpi/_dumps"
GLPI_GRAPH_DIR: "/var/lib/glpi/_graphs"
GLPI_LOCAL_I18N_DIR: "/var/lib/glpi/_locales"
GLPI_LOCK_DIR: "/var/lib/glpi/_lock"
GLPI_PICTURE_DIR: "/var/lib/glpi/_pictures"
GLPI_PLUGIN_DOC_DIR: "/var/lib/glpi/_plugins"
GLPI_RSS_DIR: "/var/lib/glpi/_rss"
GLPI_SESSION_DIR: "/var/lib/glpi/_sessions"
GLPI_TMP_DIR: "/var/lib/glpi/_tmp"
GLPI_UPLOAD_DIR: "/var/lib/glpi/_uploads"
GLPI_INVENTORY_DIR: "/var/lib/glpi/_inventories"
GLPI_NETWORK_REGISTRATION_API_URL: "https://services.glpi-network.com/api/registration/"
GLPI_MARKETPLACE_PLUGINS_API_URI: "https://services.glpi-network.com/api/glpi-plugins/"
GLPI_I18N_DIR: "/var/www/glpi/locales"
GLPI_VERSION: "10.0.7"
GLPI_SCHEMA_VERSION: "10.0.7@5d45269702917a32805e25b678f6779a98b145f6"
GLPI_MARKETPLACE_PRERELEASES: false
GLPI_MIN_PHP: "7.4.0"
GLPI_MAX_PHP: "8.3.0"
GLPI_YEAR: "2023"

</pre></details><details><summary>Libraries</summary><pre>
htmlawed/htmlawed version 1.2.9 in (/var/www/glpi/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.8.0 in (/var/www/glpi/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/glpi/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.6.2 in (/var/www/glpi/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/glpi/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/glpi/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/glpi/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/glpi/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/glpi/vendor/sabre/http/lib)
sabre/uri in (/var/www/glpi/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/glpi/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/glpi/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/glpi/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/glpi/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/glpi/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/glpi/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/glpi/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/glpi/vendor/symfony/console)
scssphp/scssphp in (/var/www/glpi/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/glpi/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/glpi/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/glpi/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/var/www/glpi/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/var/www/glpi/vendor/ramsey/uuid/src)
psr/log in (/var/www/glpi/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/glpi/vendor/psr/simple-cache/src)
psr/cache in (/var/www/glpi/vendor/psr/cache/src)
league/csv in (/var/www/glpi/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/glpi/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/glpi/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/glpi/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/glpi/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/glpi/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/glpi/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/glpi/vendor/symfony/cache)
html2text/html2text in (/var/www/glpi/vendor/html2text/html2text/src)
symfony/css-selector in (/var/www/glpi/vendor/symfony/css-selector)
symfony/dom-crawler in (/var/www/glpi/vendor/symfony/dom-crawler)
twig/twig in (/var/www/glpi/vendor/twig/twig/src)
twig/string-extra in (/var/www/glpi/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 not found
symfony/polyfill-php81 not found
symfony/polyfill-php82 in (/var/www/glpi/vendor/symfony/polyfill-php82)
league/oauth2-client in (/var/www/glpi/vendor/league/oauth2-client/src/Provider)
league/oauth2-google in (/var/www/glpi/vendor/league/oauth2-google/src/Provider)
thenetworg/oauth2-azure in (/var/www/glpi/vendor/thenetworg/oauth2-azure/src/Provider)

</pre></details><details><summary>LDAP directories</summary><pre>
Server: '*********.******.fr', Port: '389', BaseDN: 'ou=Utilisateurs,ou=Mairie,dc=******,dc=fr', Connection filter:
		'(&(objectClass=user)(objectCategory=person)(!(userAccountControl:1.2.840.113556.1.4.803:=2)))', RootDN: 'glpi@******.fr',
		Use TLS: none

</pre></details><details><summary>SQL replicas</summary><pre>
Not active

</pre></details><details><summary>Notifications</summary><pre>
Way of sending emails: SMTP+TLS (glpi@******************.***@messagerie.******************.***)

</pre></details><details><summary>Plugins list</summary><pre>
	camerainput          Name: Camera Input                   Version: 2.0.2      State: Enabled                                 
		Install Method: Marketplace
	fields               Name: Champs supplémentaires         Version: 1.20.6     State: Enabled                                 
		Install Method: Marketplace
	behaviors            Name: Comportements                  Version: 2.7.2      State: Enabled                                 
		Install Method: Marketplace
	mydashboard          Name: Dashboard                      Version: 2.1.5      State: Enabled                                 
		Install Method: Marketplace
	datainjection        Name: Data Injection                 Version: 2.13.2     State: Enabled                                 
		Install Method: Marketplace
	escalade             Name: Escalades                      Version: 2.9.0      State: Enabled                                 
		Install Method: Marketplace
	formcreator          Name: Form Creator                   Version: 2.13.6     State: Enabled                                 
		Install Method: Marketplace
	gantt                Name: gantt                          Version: 1.0.4      State: Enabled                                 
		Install Method: Marketplace
	genericobject        Name: Gestion d'objets               Version: 2.14.3     State: Enabled                                 
		Install Method: Marketplace
	manufacturersimports Name: Imports fabricants             Version: 3.0.5      State: Enabled                                 
		Install Method: Marketplace
	moreticket           Name: More ticket                    Version: 1.7.3      State: Enabled                                 
		Install Method: Marketplace
	mreporting           Name: Plus de rapports               Version: 1.8.2      State: Enabled                                 
		Install Method: Marketplace

</pre></details>

### Anything else?

we're running Glpi 10.0.7 but the bug exists for some versions now including previous 9.5.x

**Comment by Loiseau2nuit** [comment]
_>>  ... cannot work. It has probably never worked. <<_

I'm afraid to ask : is it planned to be working some day ? 😁

**Comment by Loiseau2nuit** [comment]
Hi there ! Me again !

Thanks for the fix in 10.0.13, it works pretty well but I guess I found "the" case that was not dealt with correctly.

Each reservation set to last for fixed durations like "1 day", "2 days" etc ... works fine, but, when needing a specific range of time : for instance : from Tuesday afternoon to wednesday at noon, won't recure at all, or in their own way.

![image](https://github.com/glpi-project/glpi/assets/186175/00f0cabb-8adb-41ec-a498-f5760d5c005c)

results in first occurence OK but no followings

![image](https://github.com/glpi-project/glpi/assets/186175/3d489484-4747-4396-8d76-05f9e12e0055)

So at that point I tried to make the reservation again, ticking tuesday AND wednesday, as it's a cross-day reservation, which resulted in a quite strange recursion, with errors : 

![image](https://github.com/glpi-project/glpi/assets/186175/29a288ab-f50b-44c2-8cee-020ddb70206e)

This was tested on 10.0.13.

My sysadmin is upgrading to 10.0.14 as we speak, I might be back again after that with some new elements :-)

**Comment by cedric-anne** [comment]
> Hi there ! Me again !
> 
> Thanks for the fix in 10.0.13, it works pretty well but I guess I found "the" case that was not dealt with correctly.
> 
> Each reservation set to last for fixed durations like "1 day", "2 days" etc ... works fine, but, when needing a specific range of time : for instance : from Tuesday afternoon to wednesday at noon, won't recure at all, or in their own way.
> 
> ![image](https://private-user-images.githubusercontent.com/186175/313124907-00f0cabb-8adb-41ec-a498-f5760d5c005c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3NDgxNjEsIm5iZiI6MTcxMDc0Nzg2MSwicGF0aCI6Ii8xODYxNzUvMzEzMTI0OTA3LTAwZjBjYWJiLThhZGItNDFlYy1hNDk4LWY1NzYwZDVjMDA1Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOFQwNzQ0MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hM2NkZjE1OTk1YjQxMDAxYTYyNjk1YTRmMGU1ODNmZGFiN2NiZjc2MDA1YjU3NjI4ZTdkNmE4NTQyNzU4NjQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.M02Xc5MUQp6Ne8OC3bDPVpApGOQcQ8t83i3-hs4bQlI)
> 
> results in first occurence OK but no followings
> 
> ![image](https://private-user-images.githubusercontent.com/186175/313125034-3d489484-4747-4396-8d76-05f9e12e0055.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3NDgxNjEsIm5iZiI6MTcxMDc0Nzg2MSwicGF0aCI6Ii8xODYxNzUvMzEzMTI1MDM0LTNkNDg5NDg0LTQ3NDctNDM5Ni04ZDc2LTA1ZjllMTJlMDA1NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOFQwNzQ0MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYWRkZTk0NGM5ZjQwN2ZkMGJiYWNjN2U4ZjRiYjY4ZTdkMTU2OTkxMjdjYjVhOGIwYTJkZWNlYTVjZTE3Y2IwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vEO6iEQ8LMyB8BkZHX2Y3PUbZ0qenERRep_d9qMroPc)
> 
> So at that point I tried to make the reservation again, ticking tuesday AND wednesday, as it's a cross-day reservation, which resulted in a quite strange recursion, with errors :
> 
> ![image](https://private-user-images.githubusercontent.com/186175/313127282-88881fc6-bfca-42e4-918f-9550f824daa8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3NDgxNjEsIm5iZiI6MTcxMDc0Nzg2MSwicGF0aCI6Ii8xODYxNzUvMzEzMTI3MjgyLTg4ODgxZmM2LWJmY2EtNDJlNC05MThmLTk1NTBmODI0ZGFhOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOFQwNzQ0MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZjIyYTI2OWVhYjQzMDcyOWRkOWFkZjc3YzI2Y2NiOTY5MDE1ZGQ0YjA0ZDY2NTFlMzAzMGUzYzE1ODc1ZGUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.t-HCfFy1scFxxNMbYGyrfkiIw0LtlRXVCxGkSUfweNg)
> 
> This was tested on 10.0.13.
> 
> My sysadmin is upgrading to 10.0.14 as we speak, I might be back again after that with some new elements :-)

Please open a new issue.
